app/views/admin/reports/_report_subtype.html.haml

```diff
@@ -2,4 +2,7 @@
   - report_subtypes.each do |report_subtype|
     %li
       - url = main_app.admin_report_path(report_type: report_type, report_subtype: report_subtype[1])
-      = link_to report_subtype[0], url
+      - if report_subtype[2].present? && report_subtype[2][:deprecated]
```

Comment:
```suggestion
      - if report_subtype.dig(2, :deprecated)
```

Reply:
I sometimes find myself wishing I could use the `&` operator here. But this is what `dig` is for 👍

Reply:
> I sometimes find myself wishing I could use the & operator here.

You can but it's super ugly:

```rb
report_subtype[2]&.[](:deprecated)
```

Reply:
Oh cool, I was never able to figure out how to access the `[]` method like that! But like you say, worse than using `dig`.